pages/members/[id]/index.js

```diff
@@ -1,10 +1,46 @@
-import React from 'react';
-import { useRouter } from 'next/router';
+import { getDataURL, getImgURL } from 'helper-functions/urls';
+import PropTypes from 'prop-types';
+import fetch from 'cross-fetch';
```

Comment:
NIT: `fetch` is available by default.

Reply:
If I am not importing fetch here, I was getting an error saying "Fetch is Not defined"
And that is why I imported fetch :)

Reply:
> You still need to import `isomorphic-fetch` or `isomorphic-unfetch` if you're using fetch() on the server.

My bad.